pact/Pact/Core/IR/Eval/CEK/Types.hs

```diff
@@ -228,134 +228,134 @@ data CapTokenClosure i
 
 instance NFData i => NFData (CapTokenClosure i)
 
-data CanApply (step :: CEKStepKind) (b :: K.Type) (i :: K.Type) (m :: K.Type -> K.Type)
-  = C {-# UNPACK #-} !(Closure step b i m)
```

Comment:
I assume, there was no benefit of using the pragma?

Reply:
No there wasn't, good catch. It was pointed out to me by @edmundnoble and @chessai